SimplenoteUITests/UIDs.swift

```diff
@@ -2,6 +2,7 @@
 let uidPicture_AppLogo = "simplenote-logo"
 
 let uidNavBar_AllNotes = "All Notes"
+let uidNavBar_LogIn = "Log In"
```

Comment:
I should have noticed this in the previous review but I didn't, so sorry for the out of scope suggestion. No need to address it now, just a nice-to-have tidiness idea for later.

Swift uses camelcase, there are no instances that I know of where we use `_` to separate parts of the name of a value or method. The only exception is that some testing practitioners do use `_` to separate test components in the test method name, but that's an exception and is not even a standard practice.

Anyways... What I'm trying to say is that `uidNavBar_LogIn` is not "swifty". One option could be to simply remove it and use `uidNavBarLogIn`. Another suggestion, more interesting in my opinion, would be to leverage Swift nested types feature and do this:

```swift
enum UID {

    enum NavBar {
        static let allNotes = "All Notes"
        static let logIn = "Log In"
        // ...
    }

    enum Buttons {
        static let allNotes = "All Notes"
        static let trash = "Trash"
        // ...
    }
}
```

Using an `enum` without any case means that it won't be able to instantiated (you can't do `_ = UID()`).

An extra benefit is that Xcode will help with its autocompletion:

![2021-02-22 20-29-14 2021-02-22 20_31_25](https://user-images.githubusercontent.com/1218433/108689567-57bf0480-754d-11eb-9181-ab223a2bce9d.gif)

Reply:
Thanks for bringing this up! I realized this will be discussed sooner or later, changed this to nested type as you suggested. The UIDs file now looks much tidier 🎉